streaming/base/util.py

```diff
@@ -499,11 +504,15 @@ def flaky_function():
 
     .. testoutput::
 
+        cleaning up
```

Comment:
this shoudln't be part of the output, right?

Reply:
Ah this was changed in response to @karan6181 's comments. I'll change this accordingly